guides/common/modules/ref_utilities-for-collecting-log-information.adoc

```diff
@@ -15,27 +20,34 @@ Additionally, the `foreman-debug` command exports tasks run during the last 60 d
 By default, the output tar file is located at `/tmp/__task-export-xxx.tar.xz__`.
 If the file is missing, see the file `/tmp/task-export.log` to learn why task export was unsuccessful.
 
-For more information, run `foreman-debug --help`.
+ifndef::orcharhino[]
+For more information, run `foreman-debug -h`.
+endif::[]
+ifdef::orcharhino[]
+For more information, run `orcharhino-debug -h`.
+endif::[]
 
 There is no timeout when running this command.
+ifdef::satellite[]
```

Comment:
`sosreport` works on EL7 & EL8(.5). IMHO it is the recommended tool there, also for upstream. On Debian the version is too old and the needed bugfixes aren't in. So IMHO this should be:
```suggestion
ifdef::foreman-el,katello,satellite[]
```

Note that long term we want to remove `foreman-debug` altogether.

Reply:
I could not run `sosreport` on a Forklift. Does this require a plug-in? Is it installed on Foreman by default?

Reply:
It's a separate package. I don't think we install it by default but it's included in RHEL and CentOS repositories (and I assume other RHEL rebuilds do the same thing).

Reply:
OK, apparently the package is called `sos` and provides the command `sosreport`.